examples/arduino/src/main.cpp

```diff
@@ -57,13 +63,13 @@ void setup()
   rtcos_send_event(TASK_ID_PRIORITY_ONE, EVENT_PING, (_u32)0, FALSE);
   rtcos_send_message(TASK_ID_PRIORITY_TWO, (void *)"Hello");
   u08OsTimerID = rtcos_create_timer(RTCOS_TIMER_PERIODIC, _on_os_timer_expired, 0);
```

Comment:
try passing a variable here rather than 0

Reply:
then in the timer callback try to check if the passed variable has the same value that you passed